node/src/types/deploy.rs

```diff
@@ -707,7 +711,7 @@ impl Deploy {
     ///   * the body hash is correct (should be the hash of the body), and
     ///   * approvals are non empty, and
     ///   * all approvals are valid signatures of the deploy hash
-    pub fn is_valid(&mut self) -> Result<(), DeployValidationFailure> {
+    pub fn is_valid(&mut self) -> Result<(), DeployConfigurationFailure> {
```

Comment:
Why did we change the name? I think the `Validation` was more fitting.

Reply:
We changed it cause a lot of these errors raised when a `Deploy` is checked against a specified `DeployConfig` , which is why it felt more apropos to rename it `ConfigurationFailure`

Reply:
My understanding is that we're _validating against config_ so it fails during _validation_ phase.